tests/templates/test_layers/test_basic_entangler.py

```diff
@@ -64,10 +55,9 @@ def test_rotation(self, rotation):
         weights = np.zeros(shape=(1, 2))
 
         op = qml.templates.BasicEntanglerLayers(weights, wires=range(2), rotation=rotation)
-        tape = op.expand()
+        queue = op.expand().operations
 
-        assert type(tape.operations[0]) == rotation
-        assert type(tape.operations[1]) == rotation
+        assert rotation in [type(gate) for gate in queue]
```

Comment:
This doesn't check that all gates are rotations, just that a rotation exists in that list.

Can you instead do `assert all(type(gate) == rotation for gate in queue)`?

Reply:
But the queue does not only have rotations...I basically just want to check that the custom gate type is used here..._where_ it is used is tested in the first test...